packages/assets-controllers/src/NftController.ts

```diff
@@ -378,11 +372,6 @@ export class NftController extends BaseController<
       this.#onPreferencesControllerStateChange.bind(this),
     );
 
-    this.messagingSystem.subscribe(
-      'NetworkController:networkDidChange',
-      this.#onNetworkControllerNetworkDidChange.bind(this),
-    );
```

Comment:
This subscription is removed because the purpose of `onNetworkControllerNetworkDidChange` was to populate `this.#chainId` which has been removed